index.js

```diff
@@ -557,6 +557,22 @@ exports.createFileSystem = function (volume, opts, cb) {
         else _.delayedCall(cb, S.err.NOSYS());
     }, (_n_ === '_nested_')); };
     
+    fs.createLabel = function (name, cb) {
+        fs.writeFile(name, "", () => {
+            fs.open(name, "w", (e,_fd) => {
```

Comment:
Would it be possible to use simply one `fs.open(…, "a")` call here instead nesting of `fs.write`/`fs.open`?

Reply:
Fixed